client/components/data/query-billing-transactions/README.md

```diff
@@ -24,4 +24,4 @@ export default function MyBillingTransactions( { billingTransactions } ) {
 
 ## Props
 
-- `transactionType?: 'past' | 'upcoming'`: Optionally limit the type of billing transactions returned.
```

Comment:
The server endpoint supports both `past` and `upcoming`, so the options should remain in the client code.

`/me/billing-history` returns an object with both `billing_history` and `upcoming_charges` arrays. Adding `/past` or `/upcoming` to the path limits the response to just one of the two fields.

The reducer for the `BILLING_TRANSACTIONS_RECEIVE` action has a bug that should be fixed: when a response with just one field arrives, it should keep the other one intact. But instead it removes it.

Reply:
> The server endpoint supports both past and upcoming, so the options should remain in the client code.

Done in 76436d1d4391ea4d3243d1ccd39795753d0a78ba

> The reducer for the BILLING_TRANSACTIONS_RECEIVE action has a bug that should be fixed: when a response with just one field arrives, it should keep the other one intact. But instead it removes it.

Good catch! Seems like this is an old pre-existing bug. Fixed and added a test in 0366deafd9e991b4fef60ab36ca36925da0583b5